flytekit/clients/raw.py

```diff
@@ -119,13 +119,9 @@ def __init__(self, cfg: PlatformConfig, **kwargs):
                 server_address = tuple(endpoint_parts)
             else:
                 server_address = (cfg.endpoint, "443")
-
```

Comment:
what's style guide say about this?

Reply:
I just `make fmt` and what it doesn't change is not regulated I guess.

Reply:
Is there a published flyte styleguide though?  Yes, if `fmt` addresses that's great -- also good if I can write code that the formatter doesn't need to change :-)

Reply:
No there is not unf.  Should we consider adding one?  I don't think I'd personally mind as long as it's a guide and not a strict set of rules.

Reply:
Suggesting that the guide would be the things that the formatter/linter would test for [ and reject if not adhered to ].  Anything more than that is getting too nitpicky.  I don't recall whether the project is using things like spotless, black/yapf, etc.  But, effectively a writeup of the things that these things enforce [ so people don't need to know/look-at the configurations of such tools ].

Reply:
oh yeah - `make fmt` just calls black.  and isort i think